src/fiatExchanges/UniquePaymentSection.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render } from '@testing-library/react-native'
+import * as React from 'react'
+import {
+  UniquePaymentSection,
+  UniquePaymentSectionProps,
+} from 'src/fiatExchanges/UniquePaymentSection'
+import { CICOFlow, PaymentMethod } from 'src/fiatExchanges/utils'
+import { mockProviders } from 'test/values'
+
+describe('UniquePaymentSection', () => {
+  let props: UniquePaymentSectionProps
+  beforeEach(() => {
+    props = {
+      paymentMethod: PaymentMethod.Coinbase,
+      uniqueProvider: mockProviders.filter(
+        (quote) => quote.paymentMethods[0] === PaymentMethod.Coinbase
+      )[0],
+      setNoPaymentMethods: jest.fn(),
+      flow: CICOFlow.CashIn,
+    }
+  })
+  it('shows nothing if unique provider, IE. coinbase, is restricted', async () => {
+    props.uniqueProvider!.restricted = true
+    const { queryByText } = render(<UniquePaymentSection {...props} />)
```

Comment:
probly also worth adding a test, either here or in the parent component, to make sure the `setNoPaymentMethods` callback has been used